swagger.yaml

```diff
@@ -1343,6 +1572,52 @@ components:
         - IN_PROGRESS
         - DONE
 
+    ChangeProposal:
```

Comment:
Where does this object keep the actual proposed change? E.g new task object with changed fields?